### PR TITLE
Lat trans builder

### PR DESCRIPTION
**Pull request by omodei**

### CODE REVIEW

File: threeML/plugins/FermiLATLike.py [hunk 1]
```diff
@@ -482,33 +540,36 @@ def get_log_like(self):
...
-    def getModelAndData(self):
+    def get_model_and_data(self):
 
         fake = numpy.array([])
 
         return fake, fake, fake, fake
 
     pass
 
+    def get_observation_duration(self):
```
Comment (grburgess) [inline threeML/plugins/FermiLATLike.py:570]: perhaps make a property?

**Review by grburgess — CHANGES_REQUESTED** [review]

**Comment by omodei** [comment]
All the tests now pass!

File: threeML/plugins/FermiLATLike.py [hunk 2]
```diff
@@ -176,53 +221,70 @@ def _makeFileSpectrum(self, ip):
...
+                 name,
+                 event_file,
+                 ft2_file,
+                 livetime_cube_file,
+                 kind,
+                 exposure_map_file=None,
+                 source_maps=None,
+                 binned_expo_map=None,
+                 source_name=None):
+        """FIXME! briefly describe function
```
[the remark this answers is not in the document]
Comment (omodei) [reply threeML/plugins/FermiLATLike.py]: Done

File: threeML/plugins/FermiLATLike.py (hunk 1 above)
Comment (omodei) [reply threeML/plugins/FermiLATLike.py]: I am not sure a property here would work. We really don't  want this to be set from the user. So I change the variable name to __observation_duration so it is now protected.

File: setup.cfg [hunk 3]
```diff
@@ -46,7 +46,7 @@ install_requires =
     requests
     speclite>=0.11
     ipython
-    ipyparallel
+    ipyparallel<7
```
[the remark this answers is not in the document]
Comment (ndilalla) [reply setup.cfg]: Done

**Pull request merged**